dev-tools/config.json

```diff
@@ -20,15 +20,6 @@
     "POA_RELAYER_WALLET_1": "0x34efbbc0431c7f481cdba15d65bbc9ef47196b9cf38d5c4b30afa2bcf86fafba",
     "POA_RELAYER_WALLET_2": "0xd3426cd10c4e75207bdc4802c551d21faa89a287546c2c6b3d9a0476f34934d2",
 
-    "COMMS_STORAGE_DELEGATE_1": "0x293589cdf207ed2f2253bb72b17bb7f2cfe399cdc34712b1d32908d969682238",
-    "COMMS_STORAGE_DELEGATE_2": "0x1ca1082d2304d96c2e6a3e551226e72e2cb54fddfe69b946b0efc2d9b43c19fc",
-    "COMMS_STORAGE_DELEGATE_3": "0x12712efcf90774399e272f8fc89ef264058b4cdd7f7f86956052050cbfb4350c",
-    "COMMS_STORAGE_DELEGATE_4": "0x2617e6258025c60b5aa270e02ff2247eefab37c7b463b2a870104862870ad3fb",
-
-    "COMMS_DISCOVERY_DELEGATE_1": "0xc82ad757622db5a148089e0a8fc1741cefa8677ab56a2ac9e38dac905c5ad7c7",
-    "COMMS_DISCOVERY_DELEGATE_2": "0xd2b12371a062b73ce665288f894844ab75760d0ed0c580ff19d21b54d260ceb2",
-    "COMMS_DISCOVERY_DELEGATE_3": "0x6569932dff9fbe7c04382f87ac7098f90e7aa5cb0fe636b4e5c3cb4e30668812",
-
```

Comment:
Configs should still live here

Reply:
these keys are just for comms and in docker-compose.comms.yml now because I think it's easier to see it all explicitly there without any indirection. those containers are simpler than the others - they just need the key and list of nodes on chain but nothing computed

Reply:
Can you leave a note / TODO to cleanup later, thanks

Reply:
done ✅  (in audius-compose.py since this is a JSON file)